### PR TITLE
Update first snippet

### PR DESCRIPTION
**Pull request by uliska**
This is a proof-of-example conversion. One snippet is updated to a suggested new set of metadata fields. The commits are on a new  branch and the pull request is against the `reorganization` branch - not `master`!

The suggested workflow is to create pull requests and discuss them before actually modifying the repository.

The `reorganization` branch has to be kept alive until all snippets have been moved and updated, so I suggest that we carry this on with priority because anything else that is going on on `master` in the meantime will have to be merged later too.

### CODE REVIEW

File: library/oll/lilypond-version-predicates.ily [hunk 1]
```diff
@@ -14,10 +17,11 @@
     as a three element list) as argument and
     compare that to the version number of the running LilyPond.
   }
-  % add comma-separated tags to make searching more effective:
-  tags = "Program flow, LilyPond versions"
-  % is this snippet ready?  See meta/status-values.md
+  snippet-category = "helpers"
+  tags = "program-flow,compatibility,lilypond-version"
```
Comment (fedelibre) [inline library/oll/lilypond-version-predicates.ily]: what about "workflow" instead of "program-flow" (it sounds not much meaningful to me)?

Comment (uliska) [inline library/oll/lilypond-version-predicates.ily]: Hm. I didn't mean "workflow" but maybe my wording doesn't express what I mean.
What I mean is the category that describes conditional execution or branching in programs, isn't that "program flow"? What could be better?

Comment (PeterBjuhr) [inline library/oll/lilypond-version-predicates.ily]: Control flow?? http://en.wikipedia.org/wiki/Control_flow

Comment (uliska) [inline library/oll/lilypond-version-predicates.ily]: Yes, that's it. Thanks.

**Comment by uliska** [comment]
Cool, the update (done online) changes the commit, but the discussion is still visible, although marked as outdated :-)

**Pull request merged**